perms/mutationReducer.js

```diff
@@ -10,13 +10,16 @@ module.exports = (user, perm) => {
   case types.EDIT_COMMENT:
     return true;
   case types.ADD_COMMENT_TAG:
+    return check(user, ['ADMIN', 'MODERATOR', 'STAFF']);
```

Comment:
These 3 additional lines seems unnecessary as they fall through the same result.